### PR TITLE
Add navigation to service configs

### LINKED ISSUE #3660: Add navigation config to all services
**Describe the solution you'd like**
[Add navigation config to all services](https://github.com/bbc/simorgh/tree/latest/src/app/lib/config/services).

Config: https://paper.dropbox.com/doc/Navigation-Configs--AkcytUtBnczN0gXuYyQ~x6dtAg-Jw9jAEfFkAvdKBnaIfbfr

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**
News service was omitted due to different config - issue to be created.

UKChina Trad, Zhongwen Trad, and Serbian Cyrillic were omitted pending the work on variants being completed.

WSFRONTPGS-190

### PR DESCRIPTION
**Pull request by ghost**
Resolves #3660

**Overall change:** Add navigation on all service configs

**Code changes:**

- Add navigation to service configs

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [x] This PR requires manual testing

### CODE REVIEW

File: src/app/lib/config/services/marathi.js [hunk 1]
```diff
@@ -166,6 +166,33 @@ const service = {
...
   timezone: 'Asia/Kolkata',
+  navigation: [
+    {
+      title: 'बातम्या',
+      url: '/marathi',
+    },
+    {
+      title: 'महाराष्ट्र विधानसभा निवडणूक',
+      url:
+        '/marathi/topics/efb6916d-97ac-41e3-8acf-4658102a2a0a?ocid=wsmarathi.bbc-site-referral.organic-post.topic.maharashtra-assembly-election.prod',
```
Comment (dr3) [inline src/app/lib/config/services/marathi.js]: Odd query string?

**Review by dr3 — DISMISSED** [review]
Just one tweak, i doubt we need that query string :)

File: src/app/lib/config/services/marathi.js (hunk 1 above)
Comment (ghost) [reply src/app/lib/config/services/marathi.js]: Copied from responsive news repo. Any ideas what the query string was being used for?

Comment (dr3) [reply src/app/lib/config/services/marathi.js]: I wouldnt be shocked if its a mistake tbh, maybe ask Neil or Clare?

**Review by AlistairGempf — APPROVED** [review]

**Review by DenisHdz — APPROVED** [review]
LGTM

**Review by chris-hinds — APPROVED** [review]
LGTM

File: src/app/lib/config/services/marathi.js (hunk 1 above)
Comment (staylos92) [reply src/app/lib/config/services/marathi.js]: We have definitely had some nav items added with this type of query string on purpose previously.

Comment (staylos92) [reply src/app/lib/config/services/marathi.js]: @sadickisaac I see this query string has now been removed. Has it been confirmed that it should be?

**Comment by staylos92** [comment]
LGTM 👍

**Pull request merged**